spyder/utils/clipboard_helper.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+clipboard helper module.
+"""
+from qtpy.QtWidgets import QApplication
+from qtpy.QtGui import QTextCursor
+
+from spyder.py3compat import to_text_string
+
+
+class _ClipboardHelp():
+    # Clipboard metadata
+    metadata_hash = None
+    metadata_indent = None
+    metadata_tab_stop_width_spaces = None
+
+
+def get_current_hash():
```

Comment:
Why not add the functions in this module to a class? Both the metadata attributes of the class above and these functions could be part of it.

That's unless you want to have a single instance of that class for the entire application (which seems you do).

Reply:
Yes the clipboard should be shared for the whole application as you need to save the indentation of the first line. (So you can paste from the console to the editor for example)

Reply:
Well, in that case you could create a singleton by instantiating the class I'm proposing for this at the end of the file and using it in all places you need.

That's what we do with `CONF` at the end of `spyder/config/manager.py` and the new `PLUGIN_REGISTRY` at the end of `spyder/api/plugin_registration/registry.py`